develop/plone/forms/wysiwyg.rst

```diff
@@ -120,11 +120,87 @@ You can use TinyMCE body selector make your CSS class have different styles in v
 
         Firefox does not actually support column breaks, so this was useful headaching experience.
 
+
 Customizing TinyMCE options
 ----------------------------
 
+Plone 4 uses TinyMCE 3. Plone 5 upgraded to TinyMCE 4, which works with a new concept called formats and therefore a new syntax for inline styles: `Your Custom Format's Title|custom_format_id|custom_icon_id`.
```

Comment:
Given that we have docs versioned for each Plone release, does it make sense to explain the differences here?